docs/references/javascript/organization-domain.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: OrganizationDomain
+description: The OrganizationDomain object is the model around an organization domain.
+---
+
+# `OrganizationDomain`
+
+The `OrganizationDomain` object is the model around an organization domain.
+
+## Attributes
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `id` | `string` | A unique identifier for this organization domain. |
+| `name` | `string` | The name for this organization domain (e.g example.com). |
+| `organizationId` | `string` | The organization ID of the organization this domain is for. |
+| `enrollmentMode` | `'manual_invitation' \| 'automatic_invitation' \| 'automatic_suggestion'` | An [enrollment mode][enrollment-mode-ref] will change how new users join an organization. |
+| `verification` | [`OrganizationDomainVerification`][org-domain-ver-ref] | The object that describes the status of the verification process of domain. |
+| `affiliationEmailAddress` | `string \| null` | The email address that was used to verify this organization domain. |
+| `totalPendingInvitations` | `number` | The number of total pending invitations sent to emails that match the domain name. |
+| `totalPendingSuggestions` | `number` | The number of total pending suggestions sent to emails that match the domain name. |
+| `createdAt` | `Date` | The date and time the membership was created. |
+| `updatedAt` | `Date` | The date and time the membership was updated. |

```

Comment:
Is "membership" correct? Or should it be "domain" because these are properties on the `OrganizationDomain` object?

```suggestion
| `createdAt` | `Date` | The date and time the organization domain was created. |
| `updatedAt` | `Date` | The date and time the organization domain was updated. |

```